src/app/frontend/deploy/createsecret.scss

```diff
@@ -14,6 +14,10 @@
 
 @import '../variables';
 
+.kd-create-secret-dialog {
+  min-width: 600px;
```

Comment:
Why do we need this? What was the problem? 

If we really need this can you use the 600px variable from variables.scss? the $layout-break-sm (or similar).